app/models/ApprovalProcessSummary.scala

```diff
@@ -19,7 +19,17 @@ package models
 import play.api.libs.json.{Json, OFormat}
 import java.time.LocalDate
 
-case class ApprovalProcessSummary(id: String, title: String, lastUpdated: LocalDate, status: ApprovalStatus)
+import controllers.routes
+
+case class ApprovalProcessSummary(id: String, title: String, lastUpdated: LocalDate, status: ApprovalStatus) {
+  def getSummaryPageUrl: String = {
```

Comment:
this should be in the view

Reply:
this was the bit I wanted to talk to you about as didn't feel like it should be here.  Will ping you to discuss best place